### ISSUE TITLE
Prepare mockups/designs for the sprint 1

### ISSUE THREAD

**Issue by zfrgt** [PavelLev/CourseManagementSystem#2]
In order to have decent level of product documentation, we should provide some mockups for the project before starting the estimation of the user stories. Ready-to-use webpage templates will be even better.

**Tools**: anything you can think of (e.g. google web designer, mockingbird, balsamiq mockups, etc)
**Input**: ideas on generic online course management system (e.g. udemy, coursera)
**Output**: web application mockups for all user stories of sprint 1
**Nice-to-have**: a pack of webpage templates, which you will actually use

**Comment by vasyaxparfenov** [comment]
Excuse me. Can you explain us which user story we should make mockup for? Or if we should make a mockup for whole appication?

**Comment by zfrgt** [comment]
I guess you can start with mockup for login and main screens of the application, showing all courses in a grid-like interface (this is applicable both for student and lecturer)

**Comment by zfrgt** [comment]
Here is an [example of main screen design](https://designshack.net/wp-content/uploads/inbox-card.jpg) which I found appealing. Also, here is [some kind of login form template](https://w3layouts.com/wp-content/uploads/2015/12/Portal_Login_Form.jpg) that you can use.
P.S. there is absolutely no requirement to write all the HTML/JS code on your own. You can use any template you can find on the net

**Comment by zfrgt** [comment]
Just one more thing: when you'll be sure about the web pages for main screen and login, please commit them to a separate folder of your repository - we'll have to integrate them with the backend later.

**Comment by vasyaxparfenov** [comment]
Good evening! Please check my commit, which contains trial version of our templates. Could you please share your opinion about them with us (maybe some advices) ? Thank you

**Comment by zfrgt** [comment]
Sure. Sorry for the delayed response

**Comment by vasyaxparfenov** [comment]
There is a one thing,  to open file we sent you might have to download IPlotz

**Comment by zfrgt** [comment]
That will do. Please note that Lorem ipsum under each image should contain 3 fields:
Course name
Lecturer name
Course description (very short, up to 140 characters)
